async/src/main/java/org/odk/collect/async/CoroutineScheduler.kt

```diff
@@ -49,6 +50,10 @@ open class CoroutineScheduler(private val foregroundContext: CoroutineContext, p
         throw UnsupportedOperationException()
     }
 
+    override fun networkDeferred(tag: String, spec: TaskSpec, repeatPeriod: Long, backoffPolicy: BackoffPolicy, backoffDelay: Long, inputData: Map<String, String>) {
```

Comment:
But the existing `networkDeferred` is used by `autoupdate`. I didn't want to affect that task (it wouldn't have limited retries of course but it would change it's default `backoffPolicy` and default `backoffDelay`.

Reply:
Ok, done.